sdk/python/pulumi_kubernetes/apps/v1/Deployment.py

```diff
@@ -94,13 +89,15 @@ def __init__(__self__, resource_name, opts=None, api_version=None, kind=None, me
             opts)
 
     @staticmethod
-    def get(resource_name, id, opts=None):
+    def get(resource_name: str,
+            id: pulumi.Input[str],
+            opts: Optional[pulumi.ResourceOptions] = None) -> 'Deployment':
```

Comment:
@justinvp Is the output supposed to be a literal string `'Deployment'`, or should it be `Deployment`?

Reply:
Literal string `'Deployment'` because at this point `Deployment` hasn't finished being defined yet. See https://www.python.org/dev/peps/pep-0484/#forward-references